File: Android/Martinshare/app/src/main/java/mobile/martinshare/com/martinshare/activities/MainView/EinstellungenTab/NotificationScheduleContainer.java
package mobile.martinshare.com.martinshare.activities.MainView.EinstellungenTab;

public class NotificationScheduleContainer {
    public int countA = 0, countH = 0, countS = 0;
    public NotificationScheduleContainer addToNSC(String typ) {
        if(typ.equals("a")) {
            countA++;
        } else if(typ.equals("h")) {
            countH++;
        } else if(typ.equals("s")) {
            countS++;
        }
        return this;
    }
}

File: iOS/Martinshare/Martinshare/NotificationScheduleContainer.swift
//
//  NoficationScheduleContainer.swift
//  Martinshare
//
//  Created by Modestas Valauskas on 06.02.16.
//  Copyright © 2016 ModestasV Studios. All rights reserved.
//

import Foundation
import UIKit

class NotificationScheduleContainer {
    
    var countA = 0
    var countH = 0
    var countS = 0

    func addToNSC(_ typ: String) -> NotificationScheduleContainer{
        if(typ == "a") {
            countA += 1
        } else if (typ == "h") {
            countH += 1
        } else if (typ == "s") {
            countS += 1
        }
        return self
    }

}

